src/templates/person.html

```diff
@@ -418,6 +418,15 @@ <h3 id="account"> Account for {{ person.email }} </h3>
           {{ transaction.sender_iban }}<br>
           {{ transaction.sender_bic }}
         </td>
+        <td>
+            {% if transaction.booking_type === "DIRECT_DEBIT" %}
+              <form method="POST" action="/__BACKOFFICE__/createDirectDebitReturn/{{ person.id }}/{{ transaction.id }}">
+                <button type="submit" class="btn btn-default">
+                  🔄 Create DDR
```

Comment:
🔧 I suggest we use a more explicit/clear name 🙃 (E.g: `Direct Debit Return`)

Reply:
I understand, but if I think of me as let's say a new dev, that have not seen or worked with DDR's before, it would still be easier to have something more verbose. But I am okay if we leave it as it is